packages/sdks/output/vue/nuxt.js

```diff
@@ -30,8 +30,9 @@ export default defineNuxtModule({
         };
       }
 
+      const resolver = createResolver(import.meta.url);
       addPlugin({
-        src: './nuxt-isolated-vm-plugin.js',
+        src: resolver.resolve('nuxt-isolated-vm-plugin.js'),
```

Comment:
while testing a dev release, found out that it was failing to resolve. Now works